veles/secrets/pgpass/testdata/invalid

```diff
@@ -0,0 +1,8 @@
+# password should not be equal to wildcard
+localhost:5432:mydb:myuser:*
+# port is not a number
+hostname:port:db:user:password
+# 4 groups instead of 5
+localhost:port:db:user
+# 5 groups instead of 6
+hostname:port:db:user:password:extra
```

Comment:
Add an empty newline to the end - our internal linters require that all files end in \n.
Also for the valid testdata file